CIP6/CIP6.md

```diff
@@ -0,0 +1,176 @@
+| ---           | ---                                  | ---  |
+|  `ticker`       | Pool ticker.  uppercase              | 5 Characters Maximum, Uppercase letters and numbers  |
+|  `description` | Pool Description.  Text that describes the pool | 50 Characters Maximum |
+|  `homepage` | A website URL for the pool| 64 Characters Maximum, must be a valid URL |
+|  `name` | A name for the pool | 50 Characters Maximum |
+|  `extended` | A url for extended metadata| Optional, 64 Characters Maximum, must be a valid URL |
+
+#### Extended Metadata
+The file located at the URL for extended data is a json compliant text file with the following top level fields:
+
+| key           | Description                                |
+| ---           | ---                                  |
+| `serial` | set to YYYYMMDDxx on every update |
+|  `itn` | ITN pool validation data |
+|  `info` | Corporate image, contacts and about details |
+| `pools` | Pool details |
+
+Since this extended metadata file has no checksum and can be updated at any time by the pool operator, a **serial number** is useful to easily identify updates.
+
+The purpose of this first (**itn**) extension is the verifiable linking of an ITN pool ticker with its counterpart in Mainnet to identify fraudulent duplicates.
+
+#### Complete Extended Metadata Example
+```
+{
+    "serial": 2020072001,
+    "itn": {
+        "owner": "ed25519_pk1...",
+        "witness": "ed25519_sig1..."
+    },
+    "info": {
+        "CI": {
+            "logo_url_png_icon_64x64": "https://mycoolpool.com/icon.png",
+            "logo_url_png_logo": "https://mycoolpool.com/logo.png",
+            "color_main": "#RRGGBB",
+            "logo_vector": "https://mycoolpool.com/logo.svg"
+        },
+        "social": {
+            "twitter_handle": "coolpool",
+            "telegram_handle": "coolpool",
+            "facebook_handle": "coolpool",
+            "youtube_handle": "coolpool",
+            "twitch_handle": "coolpool",
+            "discord_handle": "coolpool",
+            "github_handle": "coolpool"
+        },
+        "contact": {
+            "abuse": "abuse@pooldomain.org",
+            "support": "help@pooldomain.org",
+            "telegram_admin":"coolpool"
+        },
+        "company": {
+            "name": "Company Name",
+            "addr": "Street, Number",
+            "city": "London",
+            "country": "UK",
+            "company_id": "123456789",
+            "vat_id": "GB123456789"
+        },
+        "operator": {
+            "country": "UK",
+            "sex": "2"
+        },
+        "about": {
+            "team_affiliation": [
+                "ISPPA",
+                "Cardano Ambassador"
+            ],
+            "me": "Long description of me",
+            "server": "long description of server details",
+            "company": "long description of company details"
+        },
+        "rss": "https://mycoolpool.com/xml/poolrss.xml"
```

Comment:
RSS as well as Atom are very painful standards that are becoming obsolete, they have been discontinued by most browsers already. We might want to investigate alternatives, such as JSON Feed.

Reply:
interesting
 
indeed RSS seems has slowly died out in last 15 years (https://trends.google.com/trends/explore?date=all&q=rss)

I had to lookup what https://www.jsonfeed.org/ is. 
Exists since 2017
has surprisingly many supported plugins and libraries https://jsonfeed.org/code/

here is the github home: https://github.com/manton/JSONFeed

https://en.wikipedia.org/wiki/JSON_Feed
https://en.wikipedia.org/wiki/Comparison_of_feed_aggregators

we might need to understand first how SPos want to use news feeds (beside the social networks and chat services)